nshlib/nsh_script.c

```diff
@@ -194,7 +198,11 @@ int nsh_script(FAR struct nsh_vtbl_s *vtbl, FAR const FAR char *cmd,
 #ifdef CONFIG_NSH_ROMFSETC
 int nsh_sysinitscript(FAR struct nsh_vtbl_s *vtbl)
 {
-  return nsh_script_redirect(vtbl, "sysinit", NSH_SYSINITPATH);
+  /* Closing error log output for missing NSH_SYSINITPATH script file
```

Comment:
```suggestion
  /* Since most existing systems only use the NSH_INITPATH script file. Do not log error output for a missing 
   * NSH_SYSINITPATH script file.
```

Reply:
I think this is clearer wording. But the formatting needs to be fixed

Reply:
Done!